initialize.mjs

```diff
@@ -22,6 +22,10 @@ import {
   getSource as origGetSource,
 } from 'import-in-the-middle/hook.mjs'
 
+// Set global flag to indicate ESM loader is active
+// This is checked by rewriter.js to enable ESM rewriting for IAST
+globalThis.__DD_ESM_LOADER_ACTIVE__ = true
```

Comment:
We start to pollute the global space quite a bit. What about using a single symbol that is an object for all dd-trace-js related things we want to add to the global space? That way we limit it to that single symbol and it is actually also less visible.

To make it even less visible, I would not make it enumerable either.